ansible/roles/kstest/tasks/main.yml

```diff
@@ -0,0 +1,105 @@
+---
+
+### Install required packages
+
+- name: Install required groups
+  dnf:
+    name:
+      - "@virtualization"
+    state: latest
+
+- name: Install packages
+  dnf:
+    name:
+      - git
+      - libselinux-python
+      - NetworkManager
+      - lorax-lmc-virt # kstests livemedia-creator
+      - openssh-clients # kstests scp
+      - parallel # kstests
+      - python3-libvirt # kstests
+      - libguestfs-tools # kstests
+      - vim
+    state: latest
```

Comment:
Just from curiosity, doesn't this run multiple DNF transactions for every package one? That would result in a much slower run.
I guess it is not a problem I just wonder what will happen.

Reply:
Good point, looking here: https://github.com/ansible/ansible-modules-extras/issues/642
I think it will run just one transaction.

Reply:
Good to know :)